openedx/core/djangoapps/certificates/tests/test_api.py

```diff
@@ -67,8 +67,7 @@ def is_valid(self):
 
 @contextmanager
 def configure_waffle_namespace(feature_enabled):
-    namespace = certs_waffle.waffle()
-    auto_certificate_generation_switch = LegacyWaffleSwitch(namespace, certs_waffle.AUTO_CERTIFICATE_GENERATION)
+    auto_certificate_generation_switch = certs_waffle.AUTO_CERTIFICATE_GENERATION
     with override_waffle_switch(auto_certificate_generation_switch, active=feature_enabled):
```

Comment:
You can probably get rid of the `auto_certificate_generation_switch` variable entirely here.